lib/css/modules/_orangered.scss

```diff
@@ -25,7 +25,7 @@
 	}
 }
 
-.res-orangered-hideEmptyModmailButton {
+.res-orangered-hideEmptyModMailButton {
 	#modmail.nohavemail,
 	#modmail.nohavemail + .separator {
 	    display: none;
```

Comment:
Before 'display: none;', use a tab instead of four spaces in order to pass the lint check